Build/libHttpClient.Android/build.gradle

```diff
@@ -1,11 +1,11 @@
 apply plugin: "com.android.library"
 
 android {
-    compileSdkVersion 27
-    ndkVersion "22.0.7026061"
+    compileSdkVersion 30
+    ndkVersion '22.0.7026061'
```

Comment:
Why single quotes? I'd prefer leaving them as double, for consistency across this and other files.

Reply:
It did that by default in Android Studio, not sure why. I can change it back.